conans/test/unittests/model/ref_test.py

```diff
@@ -148,6 +148,7 @@ class CheckValidRefTest(unittest.TestCase):
     def test_string(self):
         self.assertTrue(check_valid_ref("package/1.0@user/channel"))
         self.assertTrue(check_valid_ref("package/1.0@user/channel"))
+        self.assertFalse(check_valid_ref("package/*@user/channel"))
```

Comment:
Check here the version ranges too (and with *)

Reply:
Done. Indeed the ``check_valid_ref`` is weird.

Reply:
And is that a reason to not test it or what? :P

Reply:
Nop, definitely tests were needed.
I am saying that I really want to get rid of all this mess, looking forward to Conan 2.0.

Reply:
agree